### ISSUE TITLE
E241 false negative

### ISSUE THREAD

**Issue by myint** [PyCQA/pycodestyle#71]
I recall noticing this quite a while ago.

``` python
locale_encoding_alias = {
    'ascii': 'ISO8859-1',
    'latin_1':                      'ISO8859-1',
    'iso8859_1':                    'ISO8859-1',
}
```

```
$ pep8 locale_example.py --ignore=E1,E7,E501
```

Expected:

```
locale_example.py:3:15: E241 multiple spaces after ':'
locale_example.py:4:17: E241 multiple spaces after ':'
```

Got:

```
locale_example.py:3:15: E241 multiple spaces after ':'
```
